src/sidebar/components/hooks/test/use-user-filter-options-test.js

```diff
@@ -0,0 +1,149 @@
+import { mount } from 'enzyme';
+import { createElement } from 'preact';
+
+import useUserFilterOptions from '../use-user-filter-options';
+import { $imports } from '../use-user-filter-options';
+
+describe('sidebar/components/hooks/use-user-filter-options', () => {
+  let fakeStore;
+  let lastUserOptions;
+
+  // Mount a dummy component to be able to use the hook
+  function DummyComponent() {
+    lastUserOptions = useUserFilterOptions();
+  }
+
+  function annotationFixtures() {
+    return [
+      {
+        user: 'acct:dingbat@localhost',
+        user_info: { display_name: 'Ding Bat' },
+      },
+      { user: 'abalone', user_info: { display_name: 'Aba Lone' } },
+      { user: 'bananagram', user_info: { display_name: 'Zerk' } },
+      {
+        user: 'acct:dingbat@localhost',
+        user_info: { display_name: 'Ding Bat' },
+      },
+    ];
+  }
+
+  beforeEach(() => {
+    fakeStore = {
+      allAnnotations: sinon.stub().returns([]),
+      getFocusFilters: sinon.stub().returns({}),
+      isFeatureEnabled: sinon.stub().returns(false),
+    };
+
+    $imports.$mock({
+      '../../store/use-store': { useStoreProxy: () => fakeStore },
+    });
+
+    //mount(<DummyComponent />);
+  });
+
+  afterEach(() => {
+    $imports.$restore();
+  });
+
+  it('should return a user filter option for each user who has authored an annotation', () => {
+    fakeStore.allAnnotations.returns(annotationFixtures());
+
+    mount(<DummyComponent />);
+
+    const values = lastUserOptions.map(option => option.value);
```

Comment:
You could just use one assert here and skip the `values` and `display` extraction. The same applies to the tests below.

```js
assert.deepEqual(lastUserOptions, [{ value: ..., display: ... }, { value: ..., display: ...}, ...])
```

Reply:
Aye, so I can. Changing.